tgui/packages/tgui/interfaces/GhostCrewManifest.js

```diff
@@ -0,0 +1,15 @@
+import { Section } from '../components';
+import { Window } from '../layouts';
+import { CrewManifest } from "./common/CrewManifest";
+
+export const GhostCrewManifest = (props, context) => {
```

Comment:
Final nitpick, I promise
```suggestion
export const GenericCrewManifest = (props, context) => {
```